### DISCUSSION TITLE
Can't get this running on AMD + Windows 11

### DISCUSSION THREAD

**Discussion by kazzig** [lshqqytiger/stable-diffusion-webui-directml#381] [Q&A]
Hi there,

I have big troubles getting this running on my system. Maybe some of you can lend me a hand :)

GPU:  AMD 6800XT
OS: Windows 11 Pro (10.0.22631 Build 22631)
Python Version: 3.10.6
Git installed 2.43.0 (64 bit)

Cloned: **git clone https://github.com/lshqqytiger/stable-diffusion-webui-directml && cd stable-diffusion-webui-directml && git submodule init && git submodule update** to D:/

Then running webui-user.bat leaves following error:

venv "D:\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug 1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: 1.7.0
Commit hash:
Cloning Stable Diffusion into D:\stable-diffusion-webui-directml\repositories\stable-diffusion-stability-ai...
Der Befehl ""git"" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
Traceback (most recent call last):
File "D:\stable-diffusion-webui-directml\launch.py", line 48, in
main()
File "D:\stable-diffusion-webui-directml\launch.py", line 39, in main
prepare_environment()
File "D:\stable-diffusion-webui-directml\modules\launch_utils.py", line 583, in prepare_environment
git_clone(
File "D:\stable-diffusion-webui-directml\modules\launch_utils.py", line 283, in git_clone
run(
File "D:\stable-diffusion-webui-directml\modules\launch_utils.py", line 122, in run
raise RuntimeError("\n".join(error_bits))
RuntimeError: Couldn't clone Stable Diffusion.
Command: "git" clone "https://github.com/Stability-AI/stablediffusion.git" "D:\stable-diffusion-webui-directml\repositories\stable-diffusion-stability-ai"
Error code: 1

**Comment by lshqqytiger** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/3261
